tests/timescaledb/copydb.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
+
+set -x
+set -e
+
+# This script expects the following environment variables to be set:
+#
+#  - PGCOPYDB_SOURCE_PGURI
+#  - PGCOPYDB_TARGET_PGURI
+#  - PGCOPYDB_TABLE_JOBS
+#  - PGCOPYDB_INDEX_JOBS
+
+# make sure source and target databases are ready
+pgcopydb ping 
+
+# copying roles needs superuser
+# and we use the postgres database here still
+pgcopydb copy roles 
+
+psql -o /tmp/c.out -d ${PGCOPYDB_SOURCE_PGURI} -1 -f /usr/src/pgcopydb/rides.sql
+
+copy="\COPY rides FROM nyc_data_rides.10k.csv CSV"
+psql -d ${PGCOPYDB_SOURCE_PGURI} -1 -c "${copy}"
+
+# take a snapshot using role tsdb on source database
+coproc ( pgcopydb snapshot --debug )
+
+sleep 1
+
+# copy the extensions separately, needs superuser (both on source and target)
+pgcopydb copy extensions
+
+# now clone with superuser privileges, seems to be required for timescaledb
+pgcopydb clone --skip-extensions 
+
+kill -TERM ${COPROC_PID}
+wait ${COPROC_PID}
+
+s=/tmp/fares.out
+t=/tmp/fares.out
+psql -o $s -d ${PGCOPYDB_SOURCE_PGURI} -f /usr/src/pgcopydb/fares.sql
+psql -o $t -d ${PGCOPYDB_TARGET_PGURI} -f /usr/src/pgcopydb/fares.sql
+
+diff $s $t
```

Comment:
Hopefully, last few comments!

Could you please also check the rows of the below relations are same between source and target?

timescaledb_information.hypertables
timescaledb_information.chunks

Reply:
Yes, I have checked and the rows of timescaledb_information.hypertables and timescaledb_information.chunks are same for between source and target database.

Reply:
@Kesarwani-aditi Sorry for being unclear. I meant to add checks in the copydb.sh to verify those tables has same contents.

Reply:
Could you please add this?

Reply:
Hey @arajkumar, do you think adding pgcopydb compare would do the job here?

```
pgcopydb compare schema
pgcopydb compare data
```

Reply:
@VaibhaveS I'm not sure compare schema compares extension catalog table as well. It is better to explicitly check them.

Reply:
yeah, it checks only `_timescaledb_internal._hyper_1_2_chunk` and `_timescaledb_internal._hyper_1_1_chunk` in the tests

Reply:
Added.